src/utils/CustomDeleteDialog.jsx

```diff
@@ -0,0 +1,71 @@
+import { useA11yDialog } from 'react-a11y-dialog';
```

Comment:
Wondering if this file should live in the `components` folder instead of the `utils` folder. It's my understanding that utils is more for helper functions, and not full components. Definitely not blocking, but an idea!

Reply:
I was thinking about where to put it and chose the wrong folder, I'll move it to components, thanks for the idea @michellerenehey